### PR TITLE
wal-telegram: add page

### PR DESCRIPTION
**Pull request by guillaumeboehm**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.4.0

### CODE REVIEW

File: pages/linux/wal-telegram.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Generate with wal's palette and a colored background based on the palette:
+
+`wal-telegram --tiled`
+
+- Apply a gaussian blur on the background image:
+
+`wal-telegram -g`
+
+- Specify a location for the generated theme (Default is '$XDG_CACHE_HOME/wal-telegram' or '~/.cache/wal-telegram'):
```
Comment (CleanMachine1) [inline pages/linux/wal-telegram.md:22]:
```suggestion
- Specify a location for the generated theme (default is `$XDG_CACHE_HOME/wal-telegram` or `~/.cache/wal-telegram`):
```

File: pages/linux/wal-telegram.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`wal-telegram --destination={{path/to/save/theme/in}}`
```
Comment (CleanMachine1) [inline pages/linux/wal-telegram.md:24]:
```suggestion
`wal-telegram --destination={{path/to/destination}}`
```

**Review by CleanMachine1 — COMMENTED** [review]
Sorry about the 2 days of silence!

**Review by CleanMachine1 — APPROVED** [review]

**Review by adrienthebo — APPROVED** [review]
Looks good, thanks!

**Comment by kbdharun** [comment]
@adrienthebo  , Check out <https://github.com/tldr-pages/tldr/pull/8391#issuecomment-1225909897>. We need to remove the co-authors from commit message it seems as navarroaxel mentioned. There is an ongoing PR ([Link](https://github.com/tldr-pages/tldr/pull/8398)) regarding it for updating the maintainers guide.

**Pull request merged**